wafer/users/templates/wafer.users/registration.html

```diff
@@ -0,0 +1,7 @@
+{% extends "wafer/base.html" %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% block content %}
+<h1>{% trans 'Registration:' %}</h1>
+{% crispy form %}
+{% endblock %}
```

Comment:
If the <user>/register url is going to be unconditionally accessible, this should check the status of WAFER_REGISTRATION_MODE and report something sensible for the 'ticket' case.